var/spack/repos/builtin/packages/cgdb/package.py

```diff
@@ -9,15 +9,17 @@
 class Cgdb(AutotoolsPackage):
     """A curses front-end to GDB"""
 
+    maintainers = ['tuxfan']
     homepage = 'https://cgdb.github.io'
-    url      = 'https://cgdb.me/files/cgdb-0.7.1.tar.gz'
+    url      = 'https://cgdb.me/files'
     git      = 'https://github.com/cgdb/cgdb.git'
 
-    version('master', branch='master', submodule=False, preferred=False)
+    version('master', branch='master', submodule=False, preferred=True)
```

Comment:
Do you really want to prefer an always changing development version?

Reply:
The versioned tags have a bug with newish ncurses that is fixed in `master`. I was planning to update the preferred version when a new cgdb is released.

Reply:
Please open a PR for this @ https://github.com/cgdb/cgdb/issues to ask the maintainer(s) to make a new release with the fix.

In addition, you could workaround the older releases by adding something like this after the normal ncurses depends:
```py
+    depends('ncurses@:max-or-known-good-version-which-does-not-break', when='@:lastbrokenversion')
```